src/game/g_game.pas

```diff
@@ -8220,7 +8219,10 @@ procedure g_Game_ChatSound(Text: String; Taunt: Boolean = True);
       end;
       if ok then
       begin
-        gChatSounds[i].Sound.Play();
+        // HACK: for playing taunts at volume independent from game sounds,
+        // TPlayableSound.PlayPanVolume handles a special case:
+        // Negative volume means "play this at this absolute volume" instead of relative volume.
+        gChatSounds[i].Sound.PlayPanVolume(0, (gTauntLevel / 255.0) * -1);
```

Comment:
No need to write constants as explicit reals to divide properly - we're not in C world here. ;)
```suggestion
        gChatSounds[i].Sound.PlayPanVolume(0, -(gTauntLevel / 255));
```